keyboards/idb/idb_60/readme.md

```diff
@@ -8,10 +8,19 @@ Hardware Availability: [/u/omgitspngu](https://www.reddit.com/user/omgitspngu/)
 
 Make example for this keyboard (after setting up your build environment):
 
-    make idb_60:default
+    make idb/idb_60:default
 
 Or to make and flash:
 
-    make idb_60:default:dfu
+    make idb/idb_60:default:dfu
```

Comment:
The `:flash` target will call the correct program based on the defined bootloader, and supports the most common bootloaders used within QMK.

```suggestion
    make idb/idb_60:default:flash
```